frontend/Dockerfile.e2e

```diff
@@ -3,7 +3,7 @@ FROM node:16
 RUN mkdir /srv/flagsmith && chown node:node /srv/flagsmith
 
 RUN apt-get update
-RUN wget https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_96.0.4664.110-1_amd64.deb
+RUN wget -q https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_96.0.4664.110-1_amd64.deb
```

Comment:
@kyle-ssg I think we have multiple places where the chrome version is defined? Is that right?
@matthewelwell  should move that out to an GH env var?